sympy/physics/optics/utils.py

```diff
@@ -298,13 +329,22 @@ def deviation(incident, medium1, medium2, normal=None, plane=None):
     0
     >>> deviation(r1, n1, n2, plane=P)
     -acos(-sqrt(-2*n1**2/(3*n2**2) + 1)) + acos(-sqrt(3)/3)
-
+    >>> round(deviation(0.1, 1.2, 1.5), 5)
+    -0.02005
     """
     refracted = refraction_angle(incident,
                                  medium1,
                                  medium2,
                                  normal=normal,
                                  plane=plane)
+    try:
+        angle_of_incidence = float(incident)
```

Comment:
Should you be using Float instead and returning the same are does the whole optics module default to floats when using literal input?